<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareOtbDialog</class>
 <widget class="QDialog" name="CompareOtbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare OTB Files</string>
  </property>
  <widget class="QPlainTextEdit" name="resultTextBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>152</y>
     <width>359</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton1">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>24</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton2">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>73</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file1Text">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>25</y>
     <width>278</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>35</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>File 1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file2Text">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>74</y>
     <width>278</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>58</y>
     <width>35</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>File 2:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compareButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>111</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
